cli.py

```diff
@@ -1,4 +1,4 @@
 from voicebox.__main__ import main
-
+import asyncio
 if __name__ == '__main__':
-    main()
+   asyncio.run(main())

```

Comment:
The main function is now asynchronous, which is not common for a main function. Consider refactoring the code to make the main function synchronous and move the asynchronous logic to a separate function. [medium]